Calorie Tracker/Calorie Tracker/Date.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Date.swift
+//  Calorie Tracker
+//
+//  Created by Alex Thompson on 2/25/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    static func dateToString(from date: Date) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateStyle = .medium
+        dateFormatter.timeStyle = .short
+        return dateFormatter.string(from: date)
+    }
```

Comment:
Reconsider this so that you can continue to use the same date formatter every time this is called rather that creating one every time it's called.